### PR TITLE
V2.0 init

### PR DESCRIPTION
**Pull request by maraisr**

### CODE REVIEW

**Comment by JoanClaret** [comment]
This PR includes too many changes.

As i specified in the Gitter chat :  
> i suggest you to BEMify v1 by changing selector names, keeping the same structure. So just change name selectors.

and in the contributing rules:
> Do not include unrelated commits in the same Pull Request.
Avalance + BEMify + fonts + paddings + ... = too many unrelated changes.

**Comment by JoanClaret** [comment]
I really appreciate your help, but you can't do everything by your own.

**Comment by maraisr** [comment]
You cannot just change the class names... There is so much more to BEM than just class names! There was a lot i had done, as you can see by clicking each commit, for the "deep" review. The whole reason why i asked for a v2 branch was to get community input, and not do it all on my own. But as it turns out, I am going to have to do this on my own. Frankly, I believe this project has so much potential!

Thinking maybe from here on in, I might venture off on my own and complete V2, and start its own project..

**Comment by JoanClaret** [comment]
The point is, if you want to BEMify, i'll really apreciate it. I completely agree with you that is a great thing to do.
But we should do it component by component. First the buttons. next the tabs, next ... and so on.
We should change the names and the minimum of the values (margins, paddings...) or estructure. Sometimes you BEMify and you have to change some estructure, no problem, I agree with you. We can deal with it. 

But start a completely new library is a mistake from my point of view. Change all components structure, all property values, the grid system, all variable names, and all at once: we can't deal with it.

In short, it would be awesome if you understand my point of view. 
You have an amazing SCSS skills and I would really appreciate your colaboration in BEMify the current library,  but if you want to start a new one, sadly i will said to you feel free to do it.
